### PR TITLE
GitLab Presence

### PR DESCRIPTION
**Pull request by Gurrman375**
GitLab Presence:
image
image
image
image

### CODE REVIEW

**Comment by Gurrman375** [comment]
You mean get rid of the space. Right?

**Comment by doomlerd** [comment]
I meant replace the whitespace with a dot, *not the whole exp*

**Comment by doomlerd** [comment]
still wrong

**Comment by Gurrman375** [comment]
My bad. This was the first time using a regex expression.

File: websites/G/GitLab/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "en": "GitLab is a project management and code versioning system as well as a social network platform made for developers."
+  },
+  "url": "gitlab.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/SWx8pPn.png",
+  "thumbnail": "https://i.imgur.com/ISBVR2q.png",
+  "color": "#ff4400",
+  "tags": ["programming", "sharing", "coding", "gitlab"],
+  "category": "other",
+  "regExp": ".*.gitlab.com"
```
Comment (Bas950) [inline websites/G/GitLab/dist/metadata.json:17]: `.*[.]gitlab[.]com[/]`

File: websites/G/GitLab/presence.ts [hunk 2]
```diff
@@ -0,0 +1,207 @@
+const presence = new Presence({
+    clientId: "709526684428271687"
+});
+const browsingStamp = Math.floor(Date.now() / 1000);
+let owner, title;
+presence.on("UpdateData", async() => {
+    var presenceData: presenceData = {
+        largeImageKey: "logo"
```
Comment (Bas950) [inline websites/G/GitLab/presence.ts:8]: const

**Pull request merged**